docs/source/community/persons_of_interest.rst

```diff
@@ -265,13 +298,13 @@ Windows
 -  (emeritus) Teng Gao (`gaoteng-git <https://github.com/gaoteng-git>`__)
 -  (emeritus) Peter Johnson (`peterjc123 <https://github.com/peterjc123>`__)
 
-Apple M1/MPS
-~~~~~~~~~~~~
+Apple M1/MPS/Metal
+~~~~~~~~~~~~~~~~~~~~
 
+-  Kulin Seth (`kulinseth <https://github.com/kulinseth>`__)
 -  Alban Desmaison (`alband <https://github.com/alband>`__)
 -  Nikita Shulga (`malfet <https://github.com/malfet>`__)
--  Kulin Seth (`kulinseth <https://github.com/kulinseth>`__)
--  Ramin Azarmehr (`razarmehr <https://github.com/razarmehr>`__)
+-  (emeritus) Ramin Azarmehr (`razarmehr <https://github.com/razarmehr>`__)
 
 PowerPC
```

Comment:
Do you mind adding Alfredo to emeritus (I have not seen any recent PRs from him)